website/docs/d/pi_storage_pool_capacity.markdown

```diff
@@ -47,4 +43,4 @@ In addition to all argument reference list, you can access the following attribu
 - `max_allocation_size` - (Integer) Maximum allocation storage size (GB).
 - `storage_type` - (String) Storage type of the storage pool.
 - `total_capacity` - (Integer) Total pool capacity (GB).
-- `replication_enabled` - (Boolean) Replication status of the storage pool.
```

Comment:
`replication_enabled` is sorted incorrect. I must have missed sorting it alphabetically.

It should go between `max_allocation_size` and `storage_type`

Reply:
Addressed